scripts/config.js

```diff
@@ -0,0 +1,5 @@
+export const config = {
+  firebase: {
+    /* Enter your config here */
+  }
+};
```

Comment:
FriendlyChat throws an error if config isn't provided. Want to do the same thing here?

https://github.com/firebase/codelab-friendlychat-web/blob/main/web-start/src/firebase-config.js